Changes.md

```diff
@@ -1,3 +1,8 @@
+Features
```

Comment:
Improvements. We probably should be a bit more generous with what we call a feature vs an improvement, but this is fairly small on the scale of things.